live-build/misc/migration-scripts/dx_verify

```diff
@@ -128,13 +184,19 @@ done
 
 # Where root is or will be mounted. Set in mount_datasets.
 root=
+# The path of the running dataset mounted at /var/delphix. Set in mount_datasets.
+runningVar=
+# Temporary diresctory used by the masking checks. Set in test_masking.
+masking_temp_dir=
+# The installation of java to use when executing our jars.
+java=
```

Comment:
Since the copy of java that we are using isn't coming from the new version, could we just set the value here? If the goal is to keep this script as similar as possible to the Illumos upgrade dx_verify, then I could also understand keeping it as is.

Reply:
I think that's a good idea.